src/kimmdy/assets/templates.py

```diff
@@ -17,13 +17,15 @@
 
 # Setup up your environment here
 # modules.sh might load lmod modules, set environment variables, etc.
-# source ./_modules.sh
+source ./_modules.sh
 
 CYCLE={config.max_hours}
+CYCLE_buffered=$(echo "scale=2; $CYCLE - 0.08" | bc)
+
 
 START=$(date +"%s")
 
-kimmdy --checkpoint {config.out}/kimmdy.cpt
+timeout ${{CYCLE_buffered}}h kimmdy
```

Comment:
oh, this is clever. I should add this to my own submit scripts.

Reply:
:)